blockchain/blockchain-wallet-v4-frontend
# Feat/address-labels

## Description
1. Adds label input to Import Address form
2. Reformatting btc/bch imported addresses table
3. Adding labels to send dropdowns
4. Simplifying bch dropdown selector 
5. Small changes to relevant actions/sagas/tests 

## Testing Steps
1. Generate a bitcoin private key
2. Import Address in Settings->btc->imported-addresses (notice the new form input) 
3. Actions->edit-label (notice label change)
4. Send->bch->from (notice the labels in dropdown)
